client/src/Component/MainTable.js

```diff
@@ -0,0 +1,90 @@
+import React, { useState, useEffect } from "react";
+import "./MainTableStyle.css";
+
+const MainTable = () => {
+	const [clientData, setClientData] = useState(null);
+	const [error, setError] = useState(null);
+
+	useEffect(() => {
+		fetch("/api/clients")
+			.then((response) => {
+				if (!response.ok) {
+					throw Error("We could not fetch the data for that resource");
+				}
+				return response.json();
+			})
+			.then((data) => {
+				setClientData(data);
+				setError(null);
+			})
+			.catch((err) => {
+				setError(err.message);
+				if (error != null) {
+					return <alert>{err.message}</alert>;
+				}
+			});
+	}, []);
+
+	if (clientData === null) {
+		return (
+			<div className="d-flex justify-content-center">
+				<div className="spinner-grow">
+					<span className="visually-hidden">Loading...</span>
+				</div>
+			</div>
+		);
+	}
+	return (
+		<div className="table-responsive{-xxl}">
```

Comment:
```suggestion
		<div className="table-responsive-xxl">
```
The classname is `table-responsive-xxl` so let's change it to that. Classnames usually don't have brackets in them so you can usually tell something's maybe wrong if you see them